1-A/bqelibari/bqelibari.txt

```diff
@@ -0,0 +1,153 @@
+		-nicht unterbrechende Schedulingstrategien.
+
+	-Unterbrechende Scheduler:
+		-teilen dem CPU von vorneherein nur für eine bestimmte Zeitspanne dem Prozess zu und
+		entziehen Ihn dem Prozess wieder.
+
+	-Nicht unterbrechende Scheduler:
+		Lassen den CPU solange für den Prozess laufen, bis der Prozess den CPU wieder freigibt.
+		
+		
+Init-Prozess:
+	Ist bei UNIX-Systemen der allererste Prozess mit Prozess-ID 1 (PID1), unter windows gibt es keinen vergleichbaren prozess.
+	dieser startet alle anderen Prozesse. Welche das sind und in welcher reihenfolge diese gestartet werden,
+	ist in etc/inittab festgelegt.
+	ein kill init kommt dem Herunterfahren des Systems gleich, da mit dessen Beendigung alle anderen Prozesse beendet werden.
+	Diese anderen Prozesse werden beendet da sie Kind-Prozesse des Init-Prozesses sind.
+	Als aller erstes wird vor dem Init-Prozess jedoch einen POST (Power-On-Self-Test) vom BIOS durchgeführt,
+	bei diesem wird:
+		-Funktion der CPU
+		-Funktion der RAM
+		-Funktion der Cache
+		-Funktion der I/O-Software
+	geprüft. Sollte der POST fehlschlafǵen, ertönt ein signalton und der start des Computers wird abgebrochen.
+	
+
+Shell:
+	Software die das Interagieren mit dem OS möglich macht. 
+	Kernel = Kern des OS, Shell = Hülle, für den benutzer Sichtbarer teil des OS.
+	The visible part of the OS discribes the Console, the most fundamental option for HUmans to interact with an OS.
+	The shell is so powerful that there are basically no limitations, if you have sufficient permissions, the shell allows you to delete
+	the whole file system.
+	A login shell is the first process that executes under your user ID when you log in for an interactive session. 
+	Login shells typically read a file that do things like setting environment variables, whereas the virtual shell is just another name
+	for the regular shell. This was done in order to confuse ppl who want to start working with linux.
+	
+Administratorrechte Prozess:
+	sudo chmod n+s (programmpfad) bewirkt dass das programm mit den rechten der Benutzergruppe des SuperUsers
+	ausgeführt wird. Unter Linux kann jeder Nutzer mit seinem eigenen Passwort zum Administrator werden den man in die Datei “/etc/sudoers” einträgt,
+	dieser nutzer kann dann mit seinem Passwort Adminstrative befehle ausführen. Diese nutzer sind dann berechtigt den befehl mit den rechten des Superusers
+	auszuführen. Der superuser/root-user ist das konto welches absolut keine beschränkungen hat und vollen zugriff auf jede datei im system genießt. 
+	Generell steht sudo für substitite-User do, und führt den nachfolgenden befehl mit Administratorrechten aus(falls vorhanden). 
+	Ein standardbenutzer kann keine administrativen befehlen ausführen ohne in etc/sudoers eingetragen worden zu sein.
+	
+
+.EXE Datei und ihre Struktur:
+	Bei einer .exe datei handelt es sich unter Windows-Basierten Systemen um eine Ausführbare datei.
+	Sie besteht aus:
+		-Ausführbarem Code, welcher in der .exe als .text vorliegt
```

Comment:
Clarification needed:

1. Below you state that `.text` is a certain section in an `.exe` file, but here you say that it is the executable code. What is `.text` exactly?